gcp_variant_transforms/libs/schema_converter.py

```diff
@@ -133,6 +135,13 @@ def generate_schema_from_header_fields(
       description='Unique ID (type INT64) assigned to each sample. Table with '
                   '`__sample_info` suffix contains the mapping of sample names '
                   '(as read from VCF header) to these assigned IDs.'))
+  if include_call_name:
+    calls_record.fields.append(bigquery.TableFieldSchema(
+        name=bigquery_util.ColumnKeyConstants.CALLS_NAME,
+        type=bigquery_util.TableFieldConstants.TYPE_STRING,
+        mode=bigquery_util.TableFieldConstants.MODE_NULLABLE,
+        description='Name of the call.'))
```

Comment:
How about this:
'Name of the call (sample names in the VCF Header line).'

Reply:
Done.